lib/Tools/Db/ExtendedQueryBuilder.php

```diff
@@ -592,7 +592,7 @@ public function exprLimitInArray(string $field, array $values, string $alias = '
 
 		$expr = $this->expr();
 
-		return (string)$expr->in($field, $this->createNamedParameter($values, IQueryBuilder::PARAM_STR_ARRAY));
+		return (string)$expr->in($field, $this->createNamedParameter($values, IQueryBuilder::PARAM_INT_ARRAY));
```

Comment:
TODO I should probably create a new method for this instead of overwriting this method

Reply:
Yes, definitely :)

Reply:
and put back `PARAM_STR_ARRAY` !

Reply:
Why? We are matching against integers in the database for the file id